uint64 id # The ID of the statement to delete.
---
